### PR TITLE
Doc 2963 tas manifest override

### PR DESCRIPTION
**Pull request by roshnisarangadharan**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: docs/continuous-delivery/deploy-srv-diff-platforms/tanzu/override-tanzu-manifest-and-config-files.md [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+```mdx-code-block
+import Tabs3 from '@theme/Tabs';   
+import TabItem3 from '@theme/TabItem';
+```
+```mdx-code-block
+<Tabs3>
+<TabItem3 value="Within a pipeline" label="Within a pipeline" default>
+```
+
+To override config files for all services used with an environment, do the following.
```
Comment (brian-f-harness) [inline docs/continuous-delivery/deploy-srv-diff-platforms/tanzu/override-tanzu-manifest-and-config-files.md:73]:
```suggestion
To override config files for all services used with an environment, do the following:
```

File: docs/continuous-delivery/deploy-srv-diff-platforms/tanzu/override-tanzu-manifest-and-config-files.md [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+2. Edit an existing environment or add **+ New Environment**.
+3. In **Environment Overrides**, select **+ New Config File Override**.
+4. Follow the same steps as you would when adding a config file in a service, and then select **Submit**.
+
+```mdx-code-block
+</TabItem3>
+<TabItem3 value="At an environment level" label="At an environment level">
+```
+
+To override the config files of specific services' deployed to an environment, do the following.
```
Comment (brian-f-harness) [inline docs/continuous-delivery/deploy-srv-diff-platforms/tanzu/override-tanzu-manifest-and-config-files.md:85]:
```suggestion
To override the config files of specific services deployed to an environment, do the following:
```

File: docs/continuous-delivery/deploy-srv-diff-platforms/tanzu/override-tanzu-manifest-and-config-files.md [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+6. Follow the steps to select the override file, and then select **Submit**.
+
+```mdx-code-block
+</TabItem3>    
+</Tabs3>
+```
+
+## Override variables 
+
+You can override service variables from inside a pipeline or from an environment level. When the service is deployed to that environment, the environment's variables will override the service's variables.
```
Comment (brian-f-harness) [inline docs/continuous-delivery/deploy-srv-diff-platforms/tanzu/override-tanzu-manifest-and-config-files.md:104]:
```suggestion
You can override service variables from inside a pipeline or from an environment level. When the service is deployed to that environment, the environment's variables override the service's variables.
```

File: docs/continuous-delivery/deploy-srv-diff-platforms/tanzu/override-tanzu-manifest-and-config-files.md [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+```mdx-code-block
+import Tabs2 from '@theme/Tabs';
+import TabItem2 from '@theme/TabItem';
+```
+```mdx-code-block
+<Tabs2>
+<TabItem2 value="Within a pipeline" label="Within a pipeline">
+```
+
+To override the variables for all services used with an environment, do the following.
```
Comment (brian-f-harness) [inline docs/continuous-delivery/deploy-srv-diff-platforms/tanzu/override-tanzu-manifest-and-config-files.md:115]:
```suggestion
To override the variables for all services used with an environment, do the following:
```

File: docs/continuous-delivery/deploy-srv-diff-platforms/tanzu/override-tanzu-manifest-and-config-files.md [hunk 5]
```diff
@@ -0,0 +1,141 @@
...
+4. In **New Variable**, select a variable type, then enter a name for the variable.
+5. Enter a variable value.
+6. Select **Save**.
+
+```mdx-code-block
+</TabItem2>
+<TabItem2 value="At an environment level" label="At an environment level">
+```
+
+You can override the variables of specific services' deployed to an environment.
```
Comment (brian-f-harness) [inline docs/continuous-delivery/deploy-srv-diff-platforms/tanzu/override-tanzu-manifest-and-config-files.md:129]:
```suggestion
You can override the variables of specific services deployed to an environment.
```

**Review by brian-f-harness — DISMISSED** [review]
lgtm. Added a few suggestions, please ignore them if they do not apply.

File: docs/continuous-delivery/deploy-srv-diff-platforms/tanzu/override-tanzu-manifest-and-config-files.md (hunk 5 above)
Comment (roshnisarangadharan) [reply docs/continuous-delivery/deploy-srv-diff-platforms/tanzu/override-tanzu-manifest-and-config-files.md:129]: good catch, thanks!

File: docs/continuous-delivery/deploy-srv-diff-platforms/tanzu/override-tanzu-manifest-and-config-files.md [hunk 6]
```diff
@@ -0,0 +1,141 @@
...
+```
+
+To override manifests for all services used with an environment, do the following:
+
+1. In a pipeline deploy stage, go to the **Environments** tab.
+2. Edit an existing environment or add **+ New Environment**.
+3. In **Environment Overrides**, select **+ New Manifest Override**.
+4. Specify a manifest type, and select **Continue**. 
+   
+   You can select Values YAML, OpenShift Param, Kustomize Patches, Helm Repo Overide, TAS Manifest, TAS Vars, and TAS AutoScaler manifest types.
```
Comment (RishabhGupta34) [inline docs/continuous-delivery/deploy-srv-diff-platforms/tanzu/override-tanzu-manifest-and-config-files.md:31]: Tas supports only `TAS Manifest, TAS Vars, and TAS AutoScaler manifest type`

File: docs/continuous-delivery/deploy-srv-diff-platforms/tanzu/override-tanzu-manifest-and-config-files.md [hunk 7]
```diff
@@ -0,0 +1,141 @@
...
+
+   ![manifest override](./static/manifest-override.png)
+
+5. Follow the same steps as you would when adding a manifest in a service, and then select **Submit**.
+
+```mdx-code-block
+</TabItem>
+<TabItem value="At an environment level" label="At an environment level">
```
Comment (RishabhGupta34) [inline docs/continuous-delivery/deploy-srv-diff-platforms/tanzu/override-tanzu-manifest-and-config-files.md:39]: At environment level it should highlight environment overrides, but it's highlighting service overrides

**Review by RishabhGupta34 — APPROVED** [review]

**Pull request merged**